src/implementations/vanilla/components/date-block/date-block.scss

```diff
@@ -87,7 +87,7 @@ $circle-width: 0.6rem;
   padding-bottom: map.get(spacing.$spacing, '2xs');
   padding-left: map.get(spacing.$spacing, '2xs');
   padding-right: map.get(spacing.$spacing, '2xs');
-  text-decoration: none !important;
+  text-decoration: none;
```

Comment:
it seems that this `important` was needed; there is an underline on the component now.
So we should bring it back, or increase the specificity of the rule